src/utils/time.ts

```diff
@@ -136,6 +158,15 @@ export function getDaysAgoInSeconds(days: number) {
   return getHoursAgoInSeconds(days * 24);
 }
 
+/**
+ * Format UTC seconds as a string representing time in the format hh:mm
+ *
+ * @example
+ *   formatHourAndMinute(1618509600)
+ *   // '21:00'
+ *
+ * @param utcSeconds
+ */
 export const formatHourAndMinute = (utcSeconds: number) => {
   const date = new Date(utcSeconds * ONE_SECOND_IN_MILLISECONDS);
   const hh = String(date.getHours()).padStart(2, "0");
```

Comment:
[NB] Can we also use https://date-fns.org/docs/format for this by chance?

Reply:
refactored utils/time https://github.com/sparkletown/sparkle/pull/1328/commits/bec9ea233a7130a836079592bff2df7aae3a3734